redhat-developer/intellij-openshift-connector
# feat: Create the "Happy Path" test for the "Openshift Connector" plugin

Create the "Happy Path" test for the "Openshift Connector" plugin

##Scenario:
 - Import sample project
 - Go to the "Openshift Connector" tab
 - Expand tree
 - Create deployment using imported sample
 - Check that component is successfully deployed



https://user-images.githubusercontent.com/22724619/151194621-ccea37e5-f64e-4f9a-ad1d-fd8cabbd29cc.mp4
